README.md

```diff
@@ -2,99 +2,134 @@
 
 ⚠️ This repository is in beta development ⚠️
 
-Official Klaviyo React Native SDK
+The Klaviyo React Native SDK allows developers to incorporate Klaviyo analytics and push notification functionality in
+their React Native applications for Android and iOS. It is a Typescript wrapper (native module bridge) around the native
+Klaviyo iOS and Android SDKs. For more information on the native SDKs, please see the
+[iOS](https://github.com/klaviyo/klaviyo-swift-sdk) and [Android](https://github.com/klaviyo/klaviyo-android-sdk)
+repositories. The SDK assists in identifying users and tracking user events via the latest Klaviyo Client APIs.
+To reduce performance overhead, API requests are queued and sent in batches. The queue is persisted to local storage
+so that data is not lost if the device is offline or the app is terminated.
+
+Once integrated, your marketing team will be able to better understand your app users' needs and send them timely
+push notifications via FCM (Firebase Cloud Messaging) and APNs (Apple Push Notification Service).
+
+## Requirements
+For initial beta release, the SDK was developed and tested against the latest release of React Native (0.73.1).
```

Comment:
```suggestion
For initial beta release, the SDK was developed and tested against the latest release of React Native available at the time the project was created (0.73.1).
```
or something like that

Reply:
How about "latest minor release (0.73)" ?